website/content/en/docs/scorecard/scorecard.md

```diff
@@ -685,8 +685,8 @@ Once done, follow the steps in this [document][olm-deploy-operator] to bundle yo
 - As of now, using the scorecard with a CSV does not permit multiple CR manifests to be set through the CLI/config/CSV annotations. You will have to tear down your operator in the cluster, re-deploy, and re-run the scorecard for each CR being tested. In the future the scorecard will fully support testing multiple CR's without requiring users to teardown/standup each time.
```

Comment:
Since this is now the only file in the directory, I suggest that you put this content in `_index.md`.  Otherwise, the url is redundant `scorecard/scorecard/`.

Reply:
Done